### ISSUE TITLE
Acquia cloud workflow log and deploy:update in cloud hooks

### ISSUE THREAD

**Issue by rjgwiz** [acquia/blt#783]
Is anyone else having this issue where Cloud workflow logs show this message in place of cloud hook script output?:
> --TRUNCATED DUE TO LARGE SIZE--

Our hook is calling deploy:update in cloud hooks via `deploy_updates()` (https://github.com/acquia/blt/blob/8.x/scripts/cloud-hooks/functions.sh#L6)

I haven't raised a ticket with Acquia yet. The truncated message started appearing after integrating deploy:update into our hooks and should have something to do with the verbosity (https://github.com/acquia/blt/blob/8.x/scripts/cloud-hooks/functions.sh#L15). I wanted to get a feel here if anyone else has the same issue with workflow logs. Besides calling deploy_updates() we have some additional custom cloud hook scripting with ~50 lines of short echo statements.

**Comment by grasmash** [comment]
Can you share the output? I'm curious to see what's taking up so much space. I may be able to reduce default verbosity of some tasks.

**Comment by rjgwiz** [comment]
@grasmash Ok, got to the bottom of this. Was not an issue with the verbosity. Explaining what happened to share if anyone else makes the same mistake.

Long story short: our BLT project.yml listed a module to install that we did not have (Shield) because we aren't using yet. deploy:update tried to install the module, knowing the module is missing Drush asks `Would you like to download them? (y/n)`, the -y param being on says yes to everything. Drush tries to download the module, does not have write access to the Acquia server so download fails unbeknownst to Drush, Drush thinks it was downloaded and tries to enable again. Endless loop.

**Comment by rjgwiz** [comment]
This happened before with straight cloud hooks and I figured it out by running the cloud hooks manually on dev via ssh. This time the clocking/truncated due to large size timed with a BLT update, and our Dev server was half broken on Friday so I couldn't SSH on Friday. Since it was rebooted I was able to get in and see the endless loop when deploy:update was running. So yea it's an ACE/own code integrity issue, not BLT.
